res/css/views/elements/_InviteReason.scss

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2021 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+.mx_InviteReason {
+    position: relative;
+    margin-bottom: 1em;
+
+    .mx_InviteReason_reason {
+        visibility: visible;
+    }
+
+    .mx_InviteReason_toggle {
+        display: none;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
```

Comment:
Sorry if you already know that, but I've only discovered this recently. There's now a shorthand for all those properties, [`inset`](https://developer.mozilla.org/en-US/docs/Web/CSS/inset)

```suggestion
        inset: 0;
```

I believe PostCSS should do its job and replace that in the final bundle

Reply:
Though empirically PostCSS does not translate it

![image](https://user-images.githubusercontent.com/2403652/114393024-79be2600-9b91-11eb-82be-231ddeff9df7.png)


Looks like we'd need to install https://www.npmjs.com/package/postcss-inset if we wanted that

Given lack of Safari support https://developer.mozilla.org/en-US/docs/Web/CSS/inset we should not use it without postcss support.

Reply:
Changed to `inset`.

Reply:
@jryans probably best to revert that to not risk it breaking under Safari